Animation-Smoothing.md

```diff
@@ -5,16 +5,6 @@ Off | On
 ![](img/animation-smoothing/animation_smoothing_off.gif) | ![](img/animation-smoothing/animation_smoothing_on.gif)
 
 ## Animation Smoothing
-Each of these options can be disabled individually by unchecking the respective box.
+Enables animation smoothing. 
 
-### Smooth Player Animations
-
-Configures whether the player animations are smooth or not.
-
-### Smooth NPC Animations
-
-Configures whether NPC animations are smooth or not.
-
-### Smooth Object Animations
-
-Configures whether object animations are smooth or not.
+Note: Does not work when Viggora's or Ursine Chainmace is equipped.
```

Comment:
Does this really need to be noted? Is this a frequent question/complaint which we field?

Reply:
You could include that certain animations are excluded because they are broken with animation smoothing. If you want to, you could potentially mention the chainmace as an example.

Reply:
Sweet, thanks for the feedback. updated to mention it is only one of the cases as it does seem to still be excluded: 

https://github.com/runelite/runelite/blob/2e38dc40d92f8346f5ea795c7abe9c2397f0929c/runelite-client/src/main/java/net/runelite/client/plugins/animsmoothing/AnimationSmoothingPlugin.java#L79